bfsbdump

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python3
+
+"""
+Dump BlueField secure boot status information.
+"""
+
+import argparse
+import struct
+import binascii
+import os
+import sys
+import subprocess
+
+LifeCycleState = [ 'Production', 'Secure', 'Non-Secure', 'RMA' ]
+
+def GetPlatformName():
+    Out = subprocess.run(['bfhcafw', 'mcra', '0xf0014.0:16'], stdout=subprocess.PIPE)
+    DeviceId = Out.stdout[:-1].decode('utf-8')
+    if DeviceId == '0x00000211':
+        return 'BlueField1'
+    elif DeviceId == '0x00000214':
+        return 'BlueField2'
+    elif DeviceId == '0x0000021c':
+        return 'BlueField3'
```

Comment:
Maybe do this with a table? Slightly more readable in my opinion
```python
IdTable = {
    '0x00000211': 'BlueField1',
    '0x00000214': 'BlueField2',
    '0x0000021c': 'BlueField3'
}
if DeviceId in IdTable:
    return IdTable[DeviceId]
else:
    # error...
```

Reply:
Good idea!